pom.xml

```diff
@@ -85,6 +85,12 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>13.0.1</version>
+      <scope>compile</scope>
```

Comment:
I'm not sure why this needs to be added, as I don't see guava being used in any of the new files.

Does it break if you don't have this dependency or something? Are we using guava directly anywhere in this project?

Reply:
`guava` is needed for e2e test framework.

Reply:
can this be test scope then?

Reply:
`com.google.guava:guava:jar:13.0.1:compile` is a transitive dependency. I think it's safe to declare it explicitly. Not sure if it will break the BUT if I change to `test` scope.

Reply:
it's used in `cdap-formats`
```
[INFO] +- io.cdap.cdap:cdap-formats:jar:6.8.0-SNAPSHOT:compile
[INFO] |  \- com.google.guava:guava:jar:13.0.1:compile
```

Reply:
If guava is being used by a class in this project, it should be a compile dependency in the pom. If it is only used by a dependent library like cdap-formats, you should not have to add it as an explicit dependency, it should only be a transitive dependency.

Reply:
without adding it, the e2e test can't run with guava declared just in `profile`.

Reply:
because the guava from e2e is being used and causing problems? If so, can you exclude it? 

```
  <dependency>
    <groupId>io.cdap.tests.e2e</groupId>
    <artifactId>cdap-e2e-framework</artifactId>
    <version>0.0.1-SNAPSHOT</version>
    <scope>test</scope>
    <exclusions>
      <exclusion>
        ...
      <exclusion>
    </exclusions>
  </dependency>
```

Reply:
after chatting offline, can you add a comment `<!-- comment here -->` describing why this is needed